_maps/shuttles/emergency_shelter.dmm

```diff
@@ -0,0 +1,698 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"aa" = (
+/turf/template_noop,
+/area/template_noop)
+"ab" = (
+/obj/effect/spawner/structure/window/plastitanium,
+/turf/open/space/basic,
+/area/shuttle/escape)

```

Comment:
Why is this area the escape shuttle?

Reply:
Oh, I think that's meant to be a passthrough, hold on

Reply:
Oh the windows, hold on